corehq/apps/api/query_adapters.py

```diff
@@ -49,7 +49,7 @@ def __getitem__(self, item):
                 'Invalid type of argument. Item should be an instance of slice class.')
 
         for group in groups:
-            group._precomputed_active_users = group.get_user_ids()
+            group.__dict__['get_user_ids'] = group.get_user_ids()
```

Comment:
My goal in moving it in here at all was to have all of the costly things happen within the adapter, rather than in some external step, so that it's more visible, and shows up in a shell query like `GroupQuerySetAdapterCouch(domain)[0:1000]`, rather than being in some opaque outside step. So I did mean to call it—my mistake was actually in the other implementation below, where I used a lambda but shouldn't have. Also as stated in the other message, I do think setting what you expect to be a function directly to a value is not the cleanest or most predictable thing to do (cont'd in my other comment).

Reply:
gotcha - didn't realize tastypie was okay with either a callable or a value